_posts/2023-07-31-optimizing-kani.md

```diff
@@ -0,0 +1,293 @@
+| `varint::tests::checked_ops_test...`                        | 0,95                  | 4,94            | 0,55                   | 8,98           | 3,92                     |
+| `varint::tests::table_differential_test...`                 | 0,95                  | 4,93            | 0,51                   | 9,66           | 4,02                     |
+| `varint::tests::eight_byte_sequence_test...`                | 0,96                  | 4,97            | 0,52                   | 9,55           | 4,00                     |
+| `varint::tests::four_byte_sequence_test...`                 | 0,95                  | 4,95            | 0,52                   | 9,51           | 4,01                     |
+| `varint::tests::two_byte_sequence_test...`                  | 0,97                  | 4,98            | 0,52                   | 9,57           | 3,99                     |
+| `varint::tests::one_byte_sequence_test...`                  | 0,25                  | 1,32            | 0,14                   | 9,42           | 4,02                     |
+| `varint::tests::round_trip_values_test...`                  | 0,27                  | 1,38            | 0,15                   | 9,20           | 3,92                     |
+| `xdp::decoder::tests::decode_test...`                       | 0,58                  | 2,93            | 0,31                   | 9,45           | 3,94                     |
+| `message::cmsg::tests::round_trip_test...`                  | 0,37                  | 1,59            | 0,19                   | 8,36           | 3,50                     |
+| `message::cmsg::tests::iter_test...`                        | 0,77                  | 3,75            | 0,41                   | 9,14           | 3,83                     |
+| `message::msg::tests::address_inverse_pair_test...`         | 0,85                  | 4,07            | 0,43                   | 9,46           | 3,84                     |
+| `task::completion_to_tx::assign::tests::assignment_test...` | 0,36                  | 1,48            | 0,17                   | 8,70           | 3,47                     |
+|                                                             | **Total**             | **Total**       | **Total**              | **Avg**        | **Avg**                  |
+|                                                             | 22,8                  | 114,66          | 12,33                  | 9,29           | 3,90                     |
+
+GOTO binary export is now the default because it is faster when there are multiple proof harnesses in a crate, but the switch `--write-json-symbtab` activates the JSON export if you need it. The memory consumption can be sometimes higher, but can also sometimes be lowe depending on how much opportunity for sharing of identical subtrees the crate offers.
+
+## Details on the GOTO binary format
```

Comment:
This section was supposed to be foldable, right?

Reply:
I agree. it should be foldable.

Reply:
fixed